.github/ISSUE_TEMPLATES/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
```

Comment:
Add checkbox to indicate interest in working on it myself.

Reply:
done.